core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -74,7 +74,7 @@ class OpenMPInternal {
  public:
   friend class Kokkos::OpenMP;
 
-  std::mutex m_pool_mutex;
+  int m_pool_mutex;
```

Comment:
Prefer in-class member initializer
```suggestion
  int m_pool_mutex = 0;
```